lib/WebService/Stripe.pm

```diff
@@ -111,6 +116,18 @@ method update_account(Str $id, HashRef :$data!, :$headers) {
     return $self->post( "/v1/accounts/$id", $data, headers => $headers );
 }
 
+method upload_identity_document(HashRef|Str $account_id, Str $filepath) {
+    return $self->req(
+        POST FILE_UPLOADS_URL,
```

Comment:
File uploads go to `https://uploads.stripe.com/v1/files`. I think `post` always goes to the `base_url`?